src/eBayEnterprise/RetailOrderManagement/Payload/Payment/ConfirmFundsRequest.php

```diff
@@ -80,7 +84,7 @@ protected function serializeContents()
         $paymentContext = $this->getCardNumber() ?
             $this->serializePaymentContext() :
             $this->serializePaymentContextBase();
-        return $paymentContext . $this->serializeAmount();
+        return $paymentContext . $this->serializeAmount() . "<PerformReauthorization>{$this->xmlEncode($this->getPerformReauthorization())</PerformReauthorization>";
```

Comment:
to keep this clean you should return this string value from a protected `serializePerformReauthorization` method